.buildkite/scripts/steps/beats_tests.sh

```diff
@@ -33,11 +38,13 @@ run_test_for_beat(){
 #the setup scripts will do a few things that assume we're running out of elastic-agent and will break things for beats, so run before we do actual setup
 mage -l
 
-mkdir -p build
-cd build
+# mkdir -p build
+# cd build
+mkdir -p /tmp/beats-build
+pushd /tmp/beats-build
 
-git clone --filter=tree:0 git@github.com:elastic/beats.git
-cd ..
+git clone git@github.com:elastic/beats.git
```

Comment:
Might help if we do shallow clone

```suggestion 
git clone --depth=1 git@github.com:elastic/beats.git
```